packages/commonwealth/client/scripts/views/components/react_quill_editor/use_image_uploader.tsx

```diff
@@ -44,7 +44,7 @@ export const useImageUploader = ({
 
         // insert image op at the selected index
         // for some reason, must prefix with 3 spaces or else text will be truncated
-        editor.insertText(selectedIndex, `   ![image](${uploadedFileUrl})`);
+        editor.insertText(selectedIndex, `![image](${uploadedFileUrl})`);
```

Comment:
it looks like 3 spaces were here on purpose? if its not a case anymore, please modify/delete the comment above

Reply:
Comment no longer applies. Removed!